### PR TITLE
translation pt_PT

### PR DESCRIPTION
**Pull request by amaretzek**
hope I didn't misunderstood the context for some translations...

### CODE REVIEW

File: translations/harbour-barcode-pt_PT.ts [hunk 1]
```diff
@@ -0,0 +1,378 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="pt_PT">
```
Comment (monich) [inline translations/harbour-barcode-pt_PT.ts]: I would just say "pt" as long there's no separate translations for Portugal and other Portuguese-speaking countries.

Comment (monich) [reply translations/harbour-barcode-pt_PT.ts]: I can fix that myself though.

File: translations/harbour-barcode-pt_PT.ts [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+    </message>
+    <message id="history-title">
+        <source>History</source>
+        <extracomment>History page title</extracomment>
+        <translation>Historial</translation>
+    </message>
+    <message id="history-menu-clear">
+        <source>Clear</source>
+        <extracomment>Pulley menu item</extracomment>
+        <translation type="unfinished">Limpar</translation>
```
Comment (monich) [inline translations/harbour-barcode-pt_PT.ts]: Did you leave these "unfinished" attributes on purpose? I use it to mark translations that haven't been validated by a native speaker (e.g. those coming out of Google translate)

Comment (amaretzek) [reply translations/harbour-barcode-pt_PT.ts]: Nope. Didn't pay atention to it.. Damm, I have to learn GIT...

Comment (amaretzek) [reply translations/harbour-barcode-pt_PT.ts]: Well, removed pt_PT, added pt, clicked on new pullrequest, not sure this was ok...

**Comment by monich** [comment]
To avoid recording unnecessary commits in the history, the usual way to handle review issues is something like this:
```
# <edit the file>
# git add <file>
# git commit --amend
# git push -f
```
This way, a single commit (different one after each push -f) remains in the history. Of course you always need to be careful with push -f

It's mostly an advise for the future, in this particular case it's easier for me to just press "squash and merge" button in github web ui. That would do almost the same thing, except that it will [mess up](https://github.com/isaacs/github/issues/1368) the author information.

**Comment by monich** [comment]
Merged, thanks!

**Pull request merged**